docs/deployment/hasura-ddn/connector-reusability.mdx

```diff
@@ -0,0 +1,132 @@
+can deploy a single connector in `gcp-us-west2` (close to the database) and reuse it across both subgraphs.
+
+### Step 1: Team one: Initialise and build the supergraph and deploy the connector
+
+Team one, say the "Products" team, first builds their supergraph and
+[deploys an instance](/deployment/hasura-ddn/deploy-to-ddn.mdx) of the `hasura/postgres` connector, for example in the
+`gcp-us-west2` region with [region routing](/deployment/hasura-ddn/region-routing.mdx).
+
+```sh
+ddn project init
+ddn connector init -i
+```
+
+They then introspect the database, add the necessary models, commands, and relationships to their subgraph:
+
+```sh
+ddn connector introspect my_postgres
+ddn connector show-resources my_postgres
+ddn connector models add "*"
+ddn connector commands add "*"
+ddn connector relationships add "*"
+```
+
+Next, they build their supergraph and deploy the connector to Hasura DDN:
+
+```sh
+ddn supergraph build create
+ddn deploy
+```
+
+After deployment, the `.env.cloud` file will contain the read and write URLs for the deployed connector, for example:
+
+```env
+APP_MYPOSTGRES_READ_URL="https://my-postgres-connector-read.gcp-us-west2.nhost.run"
+APP_MYPOSTGRES_WRITE_URL="https://my-postgres-connector-write.gcp-us-west2.nhost.run"
+```
+
+### Step 2: Team Two: Initialize a new connector in their local project
+
+Team two, say the "Reviews" team, should initialize their own project and connector with the connection information to
+the same data source:
+
+```sh
+ddn project init
+ddn connector init -i
+```
+
+This will create a new connector in their local project. They can then introspect and add the models, commands, and
+relationships which they need for their specific use case:
+
+```sh
+ddn connector introspect my_postgres
+ddn connector show-resources my_postgres
+ddn connector models add "*"
+ddn connector commands add "*"
+ddn connector relationships add "*"
+```
+
+### Step 3: Team Two: Use Team One's deployed connector
+
+Team two should obtain the connector URLs from Team One's `.env.cloud` file and add them to their own `.env.cloud` file:
+
+```env
+APP_MYPOSTGRES_READ_URL="https://my-postgres-connector-read.gcp-us-west2.nhost.run"
+APP_MYPOSTGRES_WRITE_URL="https://my-postgres-connector-write.gcp-us-west2.nhost.run"
+```
+
+### Step 4: Team Two: Build and deploy their supergraph without building the connector
+
+Team two can then build their supergraph without rebuilding the connector and deploy it to their DDN project:
+
+```sh
+ddn supergraph build create --no-build-connectors
+ddn deploy
+```
+
+The `--no-build-connectors` flag is crucial here as it tells DDN to use the existing connector URLs rather than trying
+to deploy a new connector instance.
+
+### Step 5: Repeat for other subgraphs and projects
+
+You can repeat these steps for any other subgraphs or projects that need access to the same PostgreSQL database.
```

Comment:
```suggestion
These teams can repeat these steps for any other subgraphs or projects that need access to the same PostgreSQL database.
```

Since we're still in the hypothetical narrative and don't want users to think this is a Postgres-only feature for connectors.